hack/quickstart/init-master.sh

```diff
@@ -16,6 +16,27 @@ function usage() {
     exit 1
 }
 
+function retry_cmd() {
```

Comment:
not a fan of duplicated code. but probably not your issue or specific to this commit. i know the old script has quite some duplication already.